DXRModules/DeusEx/Classes/DXRFlagsNGPMaxRando.uc

```diff
@@ -340,11 +349,18 @@ simulated function MaxRandoValPair(out int min, out int max)
     }
 }
 
-function NewGamePlusVal(out int val, float curve, float exp, int min, int max)
+function NewGamePlusValInt(out int val, float curve, float exp, int min, int max)
+{
+    curve = 1.0 + ((curve - 1.0) * float(moresettings.newgameplus_curve_scalar) / 100.0) + (rngfn() * 0.1);
```

Comment:
We might want to ensure that a curve greater than 1 does not fall below 1, or a curve below 1 does not rise above 1, maybe we could add some tests for that in the ExtendedTests by testing with curve values that are close to 1, like 0.99 and 1.01